app/views/sprint-6/monitor/cases/action-plan-review.html

```diff
@@ -0,0 +1,265 @@
+{% extends "layout.html" %}
```

Comment:
I'm not quite sure what this file has to do with the rest of the PR. Is it related?